src/Mod/Mesh/Gui/Command.cpp

```diff
@@ -1779,8 +1779,8 @@ CmdMeshScale::CmdMeshScale()
 {
     sAppModule = "Mesh";
     sGroup = QT_TR_NOOP("Mesh");
-    sMenuText = QT_TR_NOOP("Scale...");
-    sToolTipText = QT_TR_NOOP("Scale selected meshes");
+    sMenuText = QT_TR_NOOP("Scale…");
+    sToolTipText = QT_TR_NOOP("Scales the selected meshes");

```

Comment:
```suggestion
    sToolTipText = QT_TR_NOOP("Scales the selected mesh objects");
```
Made language consistent with [docs](https://wiki.freecad.org/Mesh_Scale).
Docs: "The Mesh Scale command scales mesh objects"